app/src/androidTest/java/ch/epfl/sdp/peakar/points/ComputePOIPointsTest.java

```diff
@@ -222,6 +229,70 @@ public void loadPOIsFromFileTest() {
 
     }
 
+    /**
+     * Test that only the highest mountains within a horizontal bearing of 6° are filtered and that
+     * the mountain in the line of sight has a priority
+     */
+    @Test
+    public void filterHighestPOIsTest() {
+        Map<POIPoint, Boolean> pois = new HashMap<>();
+        Point startPoint = new Point(0,0,0);
+
+        for(int i = 0; i < 10; i++){
+            POIPoint poi = new POIPoint(""+i, (i+1)/10f, (i+1)/10f, i);
+            poi.setHorizontalBearing(startPoint);
+            poi.setVerticalBearing(startPoint);
+            pois.put(poi, false);
+        }
```

Comment:
Comments could be added above this part

Reply:
This is actually not part of my pull request, it's code that was written by @magangloff